components/Componenten/form-elements/group-filter.njk

```diff
@@ -0,0 +1,19 @@
+<div {{ className('form-group dso-filter', [type, 'dso-input-' + type], [inputIcon, 'has-feedback'], [state, 'dso-' + state], [required, 'dso-required']) }}>
+  <div class="dso-field-container">
```

Comment:
Het is nu niet mogelijk om het label op de juiste plek te tonen in `form-horizontal`. Daarom de searchbar met op de niet-toegankelijke manier inzetten en het label van het formulier naar de input van de searchbar laten verwijzen